pkg/validations/base_test.go

```diff
@@ -147,9 +174,10 @@ func TestRunValidationsIssueCorrection(t *testing.T) {
 }
 
 func TestIncompatibleChecksAreDisabled(t *testing.T) {
-	e, err := newEngine(newEngineConfigWithAllChecks())
+
+	err := intializeEngine()
 	if err != nil {
-		t.Errorf("Error creating validation engine %v", err)
+		return
```

Comment:
Why did you change this?  If the engine fails to initialize, it should be printing a error like it was before

Reply:
This was a mistake. Should of reflected similarly to the function above it. Fixed with latest change.